providers/hns/hns_roce_u.c

```diff
@@ -134,6 +134,14 @@ static struct verbs_context *hns_roce_alloc_context(struct ibv_device *ibdev,
 		}
 	}
 
+	if (hr_dev->hw_version == HNS_ROCE_HW_VER2) {
+		context->reset_state = mmap(NULL, to_hr_dev(ibdev)->page_size,
```

Comment:
@jgunthorpe, doesn't it need some kernel patch?

Reply:
ohh, i see kernel patch now.

Please don't mix random cleanups with patches which need kernel support

Reply:
Sorry for being late,  as the associated kernel-mode patch has received some suggestions, it is necessary to temporarily abandon the patch until the user-mode and kernel-mode new schemes are determined.